apps/bmd/src/data/index.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  ElectionDefinition,
+  safeParseElectionDefinition,
+} from '@votingworks/types'
+import { readFileSync } from 'fs'
+import { join } from 'path'
+
+function electionDefinitionFromFile(
+  path: string
+): ElectionDefinition & { electionData: string } {
```

Comment:
This was my shortcut for not updating `ElectionDefinition` to include `electionData` yet. I didn't want the scope creep in this PR.